### PR TITLE
refactor(app+main)!: comment out the search and everything related to resources (DSP-1291)

### PR DESCRIPTION
**Pull request by flavens**
resolves DSP-1291

### CODE REVIEW

File: src/app/main/header/header.component.html [hunk 1]
```diff
@@ -11,14 +11,15 @@ <h1 class="mat-headline phone-tablet-only">DSP</h1>
...
+        <!-- button to reactivate when dsp-app will be used as a research platform again: -->
+        <!-- <dsp-search-panel
             [projectfilter]="true"
             [advanced]="true"
             [expert]="true"
             (search)="doSearch($event)">
-        </dsp-search-panel>
-         <!-- advanced and expert search (in desktop and tablet version) -->
+        </dsp-search-panel> -->
+         <!-- to keep? > advanced and expert search (in desktop and tablet version) -->
```
Comment (kilchenmann) [inline src/app/main/header/header.component.html:22]: "to keep?" --> good point. I think we can delete it. It could be done in a separate refactoring task

**Review by kilchenmann — APPROVED** [review]

**Comment by flavens** [comment]
@kilchenmann about the 2 PRs, should we merge them into main or in a new branch, e.g. dsp-admin?

**Comment by kilchenmann** [comment]
> @kilchenmann about the 2 PRs, should we merge them into main or in a new branch, e.g. dsp-admin?

This is the question. But how could we do a release and a build from this specific branch?

**Comment by flavens** [comment]
@kilchenmann could you check the changes in the readme file? done in 52e90f2

File: README.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
-# DSP APP &mdash; Generic user interface of DaSCH Service Platform
+# DSP-ADMIN &mdash; Admin user interface of DaSCH Service Platform
```
Comment (kilchenmann) [inline README.md:1]: It could be confusing when the repo's name is `dsp-app` but the title is `dsp-admin`...

**Review by kilchenmann — APPROVED** [review]

**Comment by kilchenmann** [comment]
@flavens wouldn't it be better to mark this PR as breaking change? Because we have significantly reduced the functionality of the app. In this case we will get a new version (4.0.0) and we know, that this version is the dsp-admin version only.

**Review by kilchenmann — APPROVED** [review]

**Pull request merged**